app/rest/project-generator/src/main/java/io/syndesis/project/converter/visitor/LogStepVisitor.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.project.converter.visitor;
+
+import io.syndesis.integration.model.steps.Log;
+import io.syndesis.integration.support.Strings;
+import io.syndesis.model.integration.Step;
+import io.syndesis.model.log.LogStep;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Locale;
+import java.util.stream.Collectors;
+
+
+public class LogStepVisitor implements StepVisitor {
```

Comment:
I think this class might be obsolete now.  You should rebase off master.

Reply:
@iocanel this class is not more needed, can we remove it ?